project/Merging.scala

```diff
@@ -15,8 +15,7 @@ object Merging {
     case PathList("javax", "ws", "rs", _ @_*)                  => MergeStrategy.first
     case "version.conf"                                        => MergeStrategy.concat
     case "logback.xml"                                         => MergeStrategy.first
-    case x if x.endsWith("kotlin-stdlib.kotlin_module")        => MergeStrategy.first
-    case x if x.endsWith("kotlin-stdlib-common.kotlin_module") => MergeStrategy.first
+    case x if x.endsWith("kotlin_module")                      => MergeStrategy.first
```

Comment:
I was getting the errors below in the automation build. I decided to just generalize the check, but please let me know if this is a bad approach… I really don't know much the merging strategy.

```
19:37:59 [info] Built: /working/google/target/scala-2.13/workbench-google-assembly-0.1-22c0a38-SNAP.jar
19:37:59 [info] Jar hash: e4ba67591bc5301b5203a65e2c6c9e3205e0c770
19:38:31 [info] done compiling
19:38:56 [error] 2 error(s) were encountered during the merge:
19:39:18 [error] 2 error(s) were encountered during the merge:
19:39:18 [error] java.lang.RuntimeException: 
19:39:18 [error] Deduplicate found different file contents in the following:
19:39:18 [error]   Jar name = auto-value-1.10.2.jar, jar org = com.google.auto.value, entry target = META-INF/kotlin-stdlib-jdk7.kotlin_module
19:39:18 [error]   Jar name = kotlin-stdlib-jdk7-1.6.10.jar, jar org = org.jetbrains.kotlin, entry target = META-INF/kotlin-stdlib-jdk7.kotlin_module
19:39:18 [error] Deduplicate found different file contents in the following:
19:39:18 [error]   Jar name = auto-value-1.10.2.jar, jar org = com.google.auto.value, entry target = META-INF/kotlin-stdlib-jdk8.kotlin_module
19:39:18 [error]   Jar name = kotlin-stdlib-jdk8-1.6.10.jar, jar org = org.jetbrains.kotlin, entry target = META-INF/kotlin-stdlib-jdk8.kotlin_module